pkgs/development/libraries/bwidget/default.nix

```diff
@@ -6,7 +6,8 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "mirror://sourceforge/tcllib/bwidget-${version}.tar.gz";
-    sha256 = "0knlnpmwam74v0qa1h9gg4f32vzzz7ays2wbslflf51ilg7nw6jk";
+    #sha256 = "0knlnpmwam74v0qa1h9gg4f32vzzz7ays2wbslflf51ilg7nw6jk";
```

Comment:
Any good reason to keep the old hash around?

Reply:
Thanks. No. I will get rid of it in an updated PR shortly. Just to note, updating the hash had no effect on the R packages that depend on bwdigets, They are all still broken. The updated PR will mark them all as broken.